doc/source/gallery/matplotlib_scalebar.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When making a geospatial plot in matplotlib, you can use [maplotlib-scalebar library](https://pypi.org/project/matplotlib-scalebar/) to add a scale bar."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas as gpd\n",
+    "from matplotlib_scalebar.scalebar import ScaleBar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a ScaleBar object"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The only required parameter for creating a ScaleBar object is *dx*. This is equal to a size of one pixel in real world. Value of this parameter depends on units of your CRS:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Projected coordinate system (meters)\n",
+    "The easiest way to add a scale bar is using a projected coordinate system with meters as units. Just set `dx = 1`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nybb = gpd.read_file(gpd.datasets.get_path('nybb'))\n",
+    "nybb.to_crs(32619, inplace=True)  # Must be converted to a projected coordinate system\n",
```

Comment:
`nybb` is projected. It just uses feet. Either say that you _want_ to reproject it to meters or keep feet (former better). Also, I'd prefer `nybb = nybb.to_crs(32619)` instead of `inplace` but that is just a personal preference.